Type conversion warning on new TIFF decoder

On just released 1.18.0, both 64-bit and 32-bit:

```
C:/msys64/home/kmilos/MINGW-packages/mingw-w64-libheif/src/libheif-1.18.0/examples/decoder_tiff.cc:136:22: warning: conversion from 'toff_t' {aka 'long long unsigned int'} to 'off_t' {aka 'long int'} may change value [-Wconversion]
  136 |   if (!seekTIFF(tif, exif_offset, SEEK_SET)) {
      |                      ^~~~~~~~~~~
```

--- a/examples/decoder_tiff.cc
+++ b/examples/decoder_tiff.cc
@@ -38,7 +38,7 @@ extern "C" {
 
 #include "decoder_tiff.h"
 
-static bool seekTIFF(TIFF* tif, off_t offset, int whence) {
+static bool seekTIFF(TIFF* tif, toff_t offset, int whence) {
   TIFFSeekProc seekProc = TIFFGetSeekProc(tif);
   if (!seekProc) {
     return false;

